### PR TITLE
feat(GKE): add autopilot ipv6 quickstart

### PR DESCRIPTION
**Pull request by apeabody**
## Description

GKE Quickstart Autopilot

Cluster
- Network (dual stack)
- SubNetwork (dual stack)
- Cluster (dual stack)

App
- Deployment (hello-app)
- Service (dual stack)

## Checklist

**Readiness**

- [x] Yes, **merge** this PR after it is approved
- [ ] No, don't **merge** this PR after it is approved

**Style**

- [x] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] My sample follows the other requirements and best practices in the [Contributing
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)

**Testing**

- [x] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):

   - [x] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [x] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check

**Intended location**

- [x] Yes, this sample will be (or already is) included on cloud.google.com
      Location(s):

- [ ] No, this sample won't be included on cloud.google.com
      Reason:

**API enablement**

- [x] If the sample needs an API enabled to pass testing, I have added the service to the [Test setup file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf)

**Review**

- [x] If this sample adds a new directory, I have added codeowners to the [CODEOWNERS file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/.github/CODEOWNERS)

### CODE REVIEW

File: gke/quickstart/autopilot/main.tf [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+#     port {
+#       port        = 80
+#       target_port = kubernetes_deployment_v1.default.spec[0].template[0].spec[0].container[0].port[0].name
+#     }
+
+#     type = "LoadBalancer"
+#   }
+# }
+# [END gke_quickstart_autopilot]
```
Comment (yliaog) [inline gke/quickstart/autopilot/main.tf:179]: uncomment ?

Comment (apeabody) [reply gke/quickstart/autopilot/main.tf:179]: Thanks @yliaog - Unfortunately GKE/GCP creates Firewall resources OOB, which block the subsequent CI terraform teardown of the network resource.  We'd also probably need to annotate as an internal LB, which isn't as desirable as an example.

Looking at how we can potentially address with documentation.

**Review by yliaog — APPROVED** [review]

File: gke/quickstart/autopilot/main.tf (hunk 1 above)
Comment (yliaog) [reply gke/quickstart/autopilot/main.tf:179]: thanks

**Comment by apeabody** [comment]
Ready pending any final feedback

CC: @melissakendall

**Pull request merged**